test/backup_fetch_handler_test.go

```diff
@@ -3,39 +3,40 @@ package test
 import (
 	"github.com/stretchr/testify/assert"
 	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/testtools"
 	"testing"
 )
 
 func TestGetRestoredBackupFilesToUnwrap_SimpleFile(t *testing.T) {
 	sentinelDto := internal.BackupSentinelDto{
-		Files: NewBackupFileListBuilder().WithSimple().Build(),
+		Files: testtools.NewBackupFileListBuilder().WithSimple().Build(),
 	}
 
 	files := internal.GetRestoredBackupFilesToUnwrap(sentinelDto)
-	assert.Contains(t, files, SimplePath)
+	assert.Contains(t, files, testtools.SimplePath)
 }
 
 func TestGetRestoredBackupFilesToUnwrap_IncrementedFile(t *testing.T) {
 	sentinelDto := internal.BackupSentinelDto{
-		Files: NewBackupFileListBuilder().WithIncremented().Build(),
+		Files: testtools.NewBackupFileListBuilder().WithIncremented().Build(),
```

Comment:
NewBackupFileListBuilder used only in backup_fetch_handler. I think you can find all such places in your code by yourself.